crates/runc/src/io.rs

```diff
@@ -55,6 +54,28 @@ impl Debug for dyn Io {
     }
 }
 
+impl<T: Io> Io for Arc<T> {
```

Comment:
I didn't get why do we need this. 

`Arc<T> automatically dereferences to T (via the Deref trait), so you can call T’s methods on a value of type Arc<T>.`

This seems redundant?

Reply:
https://github.com/containerd/rust-extensions/pull/32